client/src/redux/reducers/datafiles.reducers.js

```diff
@@ -76,6 +76,8 @@ export const initialFilesState = {
     select: {},
     upload: {},
     trash: {},
+    compress: {},
+    extract: {},
```

Comment:
Seems these should be `null` initially, as the values passed to them are strings, not objects
```suggestion
    compress: null,
    extract: null,
```